tests/builtins/test_bytes.py

```diff
@@ -7,24 +7,21 @@ class BytesTests(TranspileTestCase):
 
 class BuiltinBytesFunctionTests(BuiltinFunctionTestCase, TranspileTestCase):
     functions = ["bytes"]
+    small_ints = True
+
+    def test_set(self):
+        self.assertCodeExecution("""
+            x = {66}
+            print(bytes(x))
+            """)
+
+    def test_frozenset(self):
+        self.assertCodeExecution("""
+            x = frozenset((66))
```

Comment:
Thanks for this too. I've now fixed it by making a substitution that allows TypeErrors to come in any order for any type. Please see here: https://github.com/pybee/batavia/pull/221/files#diff-e236243f137e2c49514a1debf1b6765cR10